### ISSUE TITLE
[Accessibility] Slide Viewer Page: Toolbar lacks keyboard accessibility

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#995]
**Describe the bug**
The entire toolbar on the Slide Viewer page lacks keyboard accessibility.

**To Reproduce**
1. Go the "Slides" on the nav bar
2. Open a slide
3. Click tab to navigate through the interactive components in the slide viewer page.
4. You'd notice that you can access all the components in the toolbar using tab key.

**Expected behavior**
All toolbar elements should be keyboard navigable.
